src/librustdoc/html/highlight/sample.html

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
I think it'd be better to move this into a `test` folder and not in the middle of the source code.